dpgen2/superop/prep_run_nvnmd_train.py

```
@@ -0,0 +1,286 @@
+
+    @property
+    def keys(self):
+        return self._keys
+
+
+def _prep_run_nvnmd_train(
+    train_steps,
+    step_keys,
+    prep_train_op: Type[OP],
+    run_train_op: Type[RunNvNMDTrain],
+    prep_config: dict = normalize_step_dict({}),
```

Ruff (0.11.9)

Do not perform function call `normalize_step_dict` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)